scripts/uberenv/packages/serac/package.py

```diff
@@ -87,8 +87,7 @@ class Serac(CMakePackage):
 
     # Basic dependencies
     depends_on("mpi")
-
-    depends_on("axom~openmp~fortran~raja~umpire")
+    depends_on("cmake@3.8:")
```

Comment:
Did you end up with an old version of CMake? The fact that this is a CMakePackage should add the dependency inheritantly